docs/api_reference/src/quantities.tex

```diff
@@ -146,121 +464,1387 @@
+  { Traits<T>::explicit_import } -> std::same_as<const bool>;
+  { Traits<T>::explicit_export } -> std::same_as<const bool>;
+  typename std::bool_constant<Traits<T>::explicit_import>;
+  typename std::bool_constant<Traits<T>::explicit_export>;
+};
+
+template<typename T>
+concept @\deflibconcept{quantity_like}@ =                 //
+  @\exposconceptnc{qty-like}@<T, quantity_like_traits> &&  //
+  requires {
+    typename quantity<quantity_like_traits<T>::reference, typename quantity_like_traits<T>::rep>;
+  };
+
+template<typename T>
+concept @\deflibconcept{quantity_point_like}@ =
+  @\exposconceptnc{qty-like}@<T, quantity_point_like_traits> &&  //
+  requires {
+    typename quantity_point<quantity_point_like_traits<T>::reference,
+                            typename quantity_point_like_traits<T>::point_origin,
+                            typename quantity_point_like_traits<T>::rep>;
+  };
+\end{itemdecl}
+
+\pnum
+In the following descriptions, let
+\begin{itemize}
+\item
+\tcode{Traits} be \tcode{quantity_like_traits} or \tcode{quantity_point_like_traits},
+\item
+\tcode{Q} be a type for which \tcode{Traits<Q>} is specialized,
+\item
+\tcode{qty} be an lvalue of type \tcode{const Q}, and
+\item
+\tcode{num} be an lvalue of type \tcode{const Tratis<Q>::rep}.
+\end{itemize}
+
+\pnum
+\tcode{Q} models \tcode{\exposconceptnc{qty-like}<Traits>} if and only if:
+\begin{itemize}
+\item
+\tcode{Traits<Q>::to_numerical_value(qty)} returns the numerical value\irefiev{112-01-29} of \tcode{qty}.
+\item
+\tcode{Traits<Q>::from_numerical_value(num)} returns a \tcode{Q} with numerical value \tcode{num}.
+\item
+If \tcode{Traits} is \tcode{quantity_point_like_traits},
+both numerical values are offset from \tcode{Traits<Q>::point_origin}.
+\end{itemize}
+
+\pnum
+If the following expression is \tcode{true}, the specified conversion will be explicit.
+\begin{itemize}
+\item
+\tcode{Traits<Q>::explicit_import} for the conversion from \tcode{Q} to a quantity type.
+\item
+\tcode{Traits<Q>::explicit_export} for the conversion from a quantity type to \tcode{Q}.
+\end{itemize}
+
+\rSec2[qty.types.general]{General}
+
+\pnum
+\label{term.quantity.type}
+A \defnadj{quantity}{type}
+is a type \tcode{\placeholder{Q}}
+that is a specialization of \tcode{quantity} or \tcode{quantity_point}.
+\tcode{\placeholder{Q}} represents the value of a quantity\irefiev{112-01-28}
+with \tcode{\placeholdernc{Q}::rep} as its number
+and \tcode{\placeholdernc{Q}::reference} as its reference.
+\tcode{\placeholder{Q}} is a structural type\irefcppx{temp.param}{term.structural.type}
+if \tcode{\placeholdernc{Q}::rep} is a structural type.
+
+\pnum
+Each class template defined in subclauses \ref{qty} and \ref{qty.pt}
+have data members and special members specified below, and
+have no base classes or members other than those specified.
+
+\rSec2[qty.concepts]{Quantity concepts}
+
+\begin{itemdecl}
+template<typename T>
+concept @\deflibconcept{Quantity}@ = @\unspecnc@;
```

Comment:
Why this is unspecified?

Reply:
It's not done, but is needed by their uses (the use of `\libconcept` requires a corresponding `\deflibconcept`).